app/e2e_include_upgrades.go

```diff
@@ -4,9 +4,8 @@ package app
 
 import (
 	"github.com/babylonlabs-io/babylon/app/upgrades/signetlaunch"
-	"github.com/babylonlabs-io/babylon/app/upgrades/vanilla"
 )
 
 func init() {
-	Upgrades = append(Upgrades, vanilla.Upgrade, signetlaunch.Upgrade)
+	Upgrades = append(Upgrades, signetlaunch.Upgrade)
 }
```

Comment:
Is it possible to move this file under `test/e2e` so that all e2e stuff is there but nowhere else?

Reply:
good question, let me try 
Since the Upgrades var is public it should not be a problem

Reply:
It didn't work, the `Upgrades` var inside the app package did not contain the SignetLaunch upgrade :/ when moving this file to another package 
Not sure why

Reply:
Might be because when starting the chain, the `test/e2e` package is not loaded, and therefore the `init` function is not executed. Eitherway np with me for keeping it as is. Maybe we can add some docstrings to clarify this

Reply:
agree, added a small comment to remove the file when we are done testing